docs/content/components/content/xloader/xloader-inline.demo.tsx

```diff
@@ -0,0 +1,107 @@
+import type { Venue } from '@/lib/data/venues';
+import {
+  QueryClient,
+  QueryClientProvider,
+  useQuery,
+} from '@tanstack/react-query';
+import {
+  Aside,
+  Button,
+  Card,
+  Image,
+  Inline,
+  Scrollable,
+  SectionMessage,
+  Split,
+  Stack,
+  Text,
+  XLoader,
+} from '@marigold/components';
+
+const Venues = () => {
+  const { data, status, isFetching } = useQuery<Venue[]>({
+    queryKey: ['venues'],
+    queryFn: async () => {
+      const response = await fetch(`/api/venues?delay=1500&q=`);
+      if (!response.ok) {
+        throw new Error('Failed to fetch venues');
+      }
+      return response.json();
+    },
+    refetchOnWindowFocus: false,
+  });
+
+  if (status === 'pending' || isFetching) {
+    return (
+      <div className="h-[300px] w-full">
+        <XLoader mode="inline" />
+      </div>
+    );
+  }
+
+  if (status === 'error') {
+    return (
+      <SectionMessage variant="error">
+        <SectionMessage.Title>Whoooops...</SectionMessage.Title>
+        <SectionMessage.Content>
+          Looks like something went wrong.
+        </SectionMessage.Content>
+      </SectionMessage>
+    );
+  }
+
+  return (
+    <Scrollable height="300px">
+      <Stack space={2}>
+        {data.map(v => (
+          <Card key={v.id}>
+            <Aside sideWidth="160px" space={8}>
+              <Image alt="" src={v.image} />
+              <Stack
+                space={6}
+                id="venueDetails"
+                role="region"
+                aria-live="polite"
+              >
+                <Stack>
+                  <Text weight="extrabold" fontSize="2xl">
+                    {v.name}
+                  </Text>
+                  <Text fontStyle="italic">{v.type}</Text>
+                </Stack>
+                <Stack>
+                  <Text weight="bold">Description</Text>
+                  <Text>{v.description}</Text>
+                </Stack>
+              </Stack>
+            </Aside>
+          </Card>
+        ))}
+      </Stack>
+    </Scrollable>
+  );
+};
+
+const queryClient = new QueryClient();
+
+export default () => (
+  <QueryClientProvider client={queryClient}>
+    <Stack space={2}>
+      <Inline>
+        <Text fontSize="lg" weight="bold">
+          List of Venues
+        </Text>
+        <Split />
+        <Button
+          variant="primary"
+          onPress={() =>
+            queryClient.invalidateQueries({ queryKey: ['venues'] })
+          }
+        >
+          Reload
```

Comment:
From your explanations in the check-in meeting. Maybe we should just call it Simulate page load or Simulate initial load? But maybe also not helpful.

Reply:
"trigger reload"?